src/configs/development.json

```diff
@@ -0,0 +1,11 @@
+{
+  "env": "development",
+  "log": {
+      "format": "dev",
+      "dir": "../logs"
+  },
+  "cors": {
+      "origin": true,
```

Comment:
As per documentation:

```
Boolean - set origin to true to reflect the request origin, as defined by req.header('Origin')
```

Does this mean `true` allows the Origin set in the request header?

Reply:
Yeah, you're right. Usually someone send the `Origin` header, but we should change it to the `false` to disable cors and allow every user to use our API, am I right? 🤔

Reply:
If you disable CORS, [Preflight requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#preflighted_requests) that most browsers do in some circumstances won't work at all right? I think we should keep CORS enabled always but allowing all origin. I think keeping it at `true` will do the work, but I never used [cors](https://www.npmjs.com/package/cors) npm package before 😅